### PR TITLE
Update switch.md

### PR DESCRIPTION
**Pull request by VickyA371**
- added prop for activeThumbColor

### CODE REVIEW

File: docs/switch.md [hunk 1]
```diff
@@ -95,7 +95,17 @@ Invoked when the user tries to change the value of the switch. Receives the new
...
-Color of the foreground switch grip. If this is set on iOS, the switch grip will lose its drop shadow.
+Color of the default foreground switch grip. If this is set on iOS, the switch grip will lose its drop shadow.
+
+| Type               |
+| ------------------ |
+| [color](colors.md) |
+
+---
+
+### `activeThumbColor`
```
Comment (cortinico) [inline docs/switch.md:106]: Could you link to where this is defined in React Native?

Comment (VickyA371) [reply docs/switch.md:106]: I can't find the proper documentation for this, But when I was having an issue to manage the activeThumbColor, I searched for it and found that there is a prop accepted by the Switch component and it is not in the documentations and not in the types of the Switch component.

I have raised PR for the type changes in react-native repo also.

Comment (cortinico) [reply docs/switch.md:106]: Could you link your other PR then?

Comment (VickyA371) [reply docs/switch.md:106]: How can I link my other PR?
Could you please help me with that?

Here is the PR link: https://github.com/facebook/react-native/pull/38636

Comment (cortinico) [reply docs/switch.md:106]: That's sufficient so we can merge this one once your other PR is merged

Comment (VickyA371) [reply docs/switch.md:106]: Hi @cortinico The above PR merged! 😊
Kindly check

**Pull request closed without merging**